src/app/components/event/event.component.html

```diff
@@ -1,14 +1,12 @@
 <div class="eventsOnCampus">
-    <p>
-        event works!
-    </p>
-    <p>
-        event works!
-    </p>
-    <p>
-        event works!
-    </p>
-    <p>
-        event works!
-    </p>
+    <br><br><br>
+    <h2 class="title">ALL EVENTS ARE CANCELLED !</h2>
```

Comment:
I think we need to atleast show that we can display some events in the events page, or any nearby locations of interest. I don't think just putting "All events are cancelled" is a valid solution. Maybe there is a way to get events through the google maps API itself? Please advise

Reply:
I know we dont have the calendar API working yet, I will try my best today/tomorow to get it up and running, and then from there we can atleast show the user's events. Will keep u posted on my progress integrating the API. But if there is a way to get events through the Map API, please give that a try. Thanks in advance!

Reply:
will look into it.
perfect keep me posted for the calendar API